rust/feature-flags/src/handler/session_recording.rs

```diff
@@ -41,17 +41,18 @@ pub fn session_recording_config_response(
     let minimum_duration = team.session_recording_minimum_duration_milliseconds;
 
     // linked_flag logic
-    let linked_flag = match &team.session_recording_linked_flag {
-        Some(cfg) => {
-            let key = cfg.get("key");
-            let variant = cfg.get("variant");
+    let linked_flag: Option<Value> = match &team.session_recording_linked_flag {
```

Comment:
https://posthog.slack.com/archives/C07Q2U4BH4L/p1749803999304849
`linked_flag` should be just a string if no variant
eg `"linkedFlag": "test-flag"` instead of
```
"linkedFlag": {
      "flag": "test-flag",
      "variant": null
    }
```

Reply:
it'd be consistent with /decide and the remote config API